Packs/Anomali_ThreatStream/Integrations/Anomali_ThreatStream_v2/Anomali_ThreatStream_v2.yml

```diff
@@ -1640,15 +1293,74 @@ script:
     - contextPath: URL.Tags
       description: (List) Tags of the URL.
       type: Unknown
-  dockerimage: demisto/emoji:1.0.0.19032
-  feed: false
-  isfetch: false
-  longRunning: false
-  longRunningPort: false
+  - arguments: []
+    description: Retrieves whitelist entries from the ThreatStream platform.
+    name: threatstream-get-whitelist
+    outputs:
+    - contextPath: ThreatStream.WhiteList.CreatedTS
+      description: Timestamp of when the entry was created.
+      type: string
+    - contextPath: ThreatStream.WhiteList.ID
+      description: Unique ID associated with the whitelist entry.
+      type: string
+    - contextPath: ThreatStream.WhiteList.ModifiedTS
+      description: Timestamp of when the entry was most recently modfied.
+      type: string
+    - contextPath: ThreatStream.WhiteList.Notes
+      description: Contextual note associated with the entry.
+      type: string
+    - contextPath: ThreatStream.WhiteList.Value
+      description: Value of the entry.
+      type: string
+    - contextPath: ThreatStream.WhiteList.ValueType
+      description: Type of entry—cidr, domain, email, ip, md5 url, user-agent.
+      type: string
+    - contextPath: ThreatStream.WhiteList.ResourceURI
+      description: Resource endpoint from where the data is extracted
+  - arguments:
+    - auto: PREDEFINED
+      description: Type of entry—cidr, domain, email, ip, md5, url, user-agent
+      name: value_type
+      predefined:
+      - cidr
+      - domain
+      - email
+      - ip
+      - md5
+      - url
+      - user-agent
+      required: true
+    - description: Value of the entry.
+      name: value
+      required: true
+    - description: Contextual note associated with the entry.
+      name: notes
+      required: true
+    description: Create a new whitelist entry.
+    execution: true
+    name: threatstream-add-new-whitelist-entry
+  - arguments:
+    - description: Unique ID associated with the whitelist entry.
```

Comment:
please mention here how that ID can be retrieved (i.e. `Can be retrieved by running the ... command.`)